umm-spec-lib/src/cmr/umm_spec/test/umm_record_sanitizer.clj

```diff
@@ -155,14 +243,14 @@
 
 (defn- sanitize-umm-record-urls
   "Sanitize all RelatedUrls throughout the collection by making them url strings."
-  [record]
+  [record concept]
```

Comment:
Your use of `concept` here probably is not consistent with other places is cmr.  It should be concept-type.

Reply:
I think this code could probably be cleaner and clearer if we used defmulti for our url-content-type mappings.  Not necessary here because it would require significant time for refactoring.  Maybe just add a comment saying we should consolidate and simplify this.  For now, changing `concept` to `concept-type` is fine.

Reply:
Thanks and done.